packages/runner/injection/multi-domain.js

```diff
@@ -33,7 +33,7 @@ const Cypress = findCypress()
 // This may not be needed if we defer to the first command
 if (!Cypress) {
```

Comment:
So now, since you are only injecting multi-domain into domains covered by a switchToDomain command, Is there any way we can get into this scenario, at least without something unexpected going terribly wrong?

and if that is the case, should we send this error to the spec bridge to show in the runner?

Reply:
Discussed during scrum and we are going to leave this check in as is for now.